<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="infoLabel">
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 170, 0);</string>
      </property>
      <property name="text">
       <string>command</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="hotkey">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>23</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(47, 141, 255);</string>
      </property>
      <property name="text">
       <string>Hotkey</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QPushButton" name="confirmButton">
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QTextEdit" name="commandEditor">
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;command&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>542</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirmButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setCommandDefinition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commandEditor</sender>
   <signal>textChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>setCommandInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setCommandInfo()</slot>
  <slot>setCommandDefinition()</slot>
 </slots>
</ui>
